src/ImageSharp/ColorSpaces/Conversion/ICompanding.cs

```diff
@@ -1,16 +1,14 @@
 ﻿// Copyright (c) Six Labors and contributors.
 // Licensed under the Apache License, Version 2.0.
 
-using SixLabors.ImageSharp.ColorSpaces.Conversion.Implementation.RgbColorSapce;
-
-namespace SixLabors.ImageSharp.ColorSpaces
+namespace SixLabors.ImageSharp.ColorSpaces.Conversion.Implementation
 {
     /// <summary>
     /// Pair of companding functions for <see cref="RgbWorkingSpace"/>.
     /// Used for conversion to <see cref="CieXyz"/> and backwards.
     /// See also: <seealso cref="RgbWorkingSpace.Companding"/>
     /// </summary>
-    internal interface ICompanding
+    public interface ICompanding
```

Comment:
I wonder why do we have interfaces which are not used through the abstraction. I think they shouldn't be there, and the methods should be moved to static utilities. It's actually **very** expensive to use virtual calls to convert only one `float` value.